vignettes/Data_preprocessing.Rmd

```diff
@@ -0,0 +1,69 @@
+---
+title: "Data preprocess"
+output: 
+  bookdown::html_document2:
+    base_format: rmarkdown::html_vignette
+    fig_caption: yes
+bibliography: "covid.bib"
+link-citations: yes
+vignette: >
+  %\VignetteIndexEntry{Data preprocess}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+  
+# Aim and data description
+  
+In this tutorial, we will provide a short tutorial about data pre-processing, which involves transforming raw data into the desired format and object for running the Bayesian Hierarchical Model (BHM) in the "fdmr" package. To illustrate the process, we will use COVID-19 infection data as a practical example.
+
+In the COVID-19 tutorial [https://4dmodeller.github.io/fdmr/articles/covid.html](https://4dmodeller.github.io/fdmr/articles/covid.html), we aim to fit a Bayesian spatio-temporal model to predict the COVID-19 infection rates across mainland England over space and time, and investigate the impacts of socioeconomic, demographic and environmental factors on COVID-19 infection. The study region is mainland England, which is partitioned into 6789 Middle Layer Super Output Areas (MSOAs). The raw shapefile of the study region is obtained from [https://geoportal.statistics.gov.uk/datasets](https://geoportal.statistics.gov.uk/datasets/ons::msoa-dec-2011-boundaries-super-generalised-clipped-bsc-ew-v3/explore?location=52.782096%2C-2.465779%2C7.81), which stores the location, shape and attributes of geographic features for the MSOAs.
+
+First we load in the shapefile into R, and store it in an object named "sp_data".
+
+```{r shapefile, warning=FALSE,error=FALSE}
+sp_data<-rgdal::readOGR(dsn='MSOA_(Dec_2011)_Boundaries_Super_Generalised_Clipped_(BSC)_EW_V3.shp')
+```
+
+The type of the object "sp_data" is a SpatialPolygonsDataFrame. 
+
+```{r classsp}
+class(sp_data)
+```
+
+Then we retrieve the projection attributes of the shapefile, i.e., "sp_data", and transform it from its original coordinate reference system (CRS) to a new CRS, which is the World Geodetic System 1984 (WGS84).

```

Comment:
```suggestion
Then we retrieve the projection attributes of the shapefile, i.e., `sp_data`, and transform it from its original coordinate reference system (CRS) to a new CRS, which is the World Geodetic System 1984 (WGS84).

```

Reply:
It's best to use backticks to show code / variables in Markdown.

Reply:
Thanks @gareth-j . I have now changed the code to use backticks instead of quotation marks.